### ISSUE TITLE
Admin\Operations - External User can view Operations grid column link "View facilities"

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1689]
#### Description:
As an External User, I want to see a list of the facilities that belong to an operation
[Figma link](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=7817-186&mode=design&t=I8ViKf1PvXw9raTb-4)

#### Acceptance Criteria:

Given I am on the operations grid,
I see a facilities column with a view facilities link
Given that I click on the link, then I am brought to the facilities grid for that operation
https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7763-4636&t=rBpJCzhxRzIMrjwm-4

#### Development Checklist:

- [ ] 

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

Blocked by #1360, #1701, and #1686

**Comment by shon-button** [comment]
@patriciarussellCAS

Reachable at:
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/operations

**Comment by patriciarussellCAS** [comment]
Approved
